src/lib/components/layout/Layout.component.js

```diff
@@ -1,24 +1,64 @@
 //@flow
-import React from "react";
-import styled from "styled-components";
-import type { Node } from "react";
-import { navbarHeight } from "../../style/theme";
-import Navbar from "../navbar/Navbar.component";
-import type { Props as NavbarProps } from "../navbar/Navbar.component";
-import type { Props as SidebarProps } from "../sidebar/Sidebar.component";
-import Sidebar from "../sidebar/Sidebar.component";
-import { getThemePropSelector } from "../../utils";
+import React from 'react';
+import styled, { css } from 'styled-components';
+import type { Node } from 'react';
+import { navbarHeight } from '../../style/theme';
+import Navbar from '../navbar/Navbar.component';
+import type { Props as NavbarProps } from '../navbar/Navbar.component';
+import type { Props as SidebarProps } from '../sidebar/Sidebar.component';
+import Sidebar from '../sidebar/Sidebar.component';
+import { getThemePropSelector, getTheme } from '../../utils';
 type Props = {
   navbar?: NavbarProps,
   sidebar?: SidebarProps,
   navbarElement?: Node,
-  children: Node
+  children: Node,
 };
 
 const LayoutContainer = styled.div`
   display: flex;
   flex-direction: column;
   height: 100vh;
+
+  ${(props) => {
+    const brand = getTheme(props);
+    return css`
+      // Custom scrollbar
+      * {
+        // Chrome / Safari / Edge
+        ::-webkit-scrollbar {
+          width: 8px;
+        }
+
+        ::-webkit-scrollbar-track {
+          background: ${brand.primary};
+        }
+
+        ::-webkit-scrollbar-thumb {
+          width: 4px;
+          background: ${brand.border};
+          border-radius: 10px;
+          -webkit-border-radius: 10px;
+          background-clip: padding-box;
+          border-left: 3px solid rgba(0, 0, 0, 0);
+          border-right: 1px solid rgba(0, 0, 0, 0);
+        }
+
+        ::-webkit-scrollbar-button {
+          width: 0;
+          height: 0;
+          display: none;
+        }
+        ::-webkit-scrollbar-corner {
+          background-color: transparent;
+        }
+
+        // Firefox
+        scrollbar-color: ${brand.border} ${brand.primary};
+        scrollbar-width: thin;
+      }
```

Comment:
Ok for the approach ! But I think we should add it to `/src/lib/index.css` in order to make it available to applications even if they don't use this `Layout` component (we may have this use case for embedded applications)

Reply:
@JBWatenbergScality Ok, make sense. Do you have any suggestions on how to use the color variables (in this case primary and border) in a .css file ?

Reply:
I see 2 options : 
1. Create a Scrollbar component independent from the layout and document that it should be used as a top level wrapper if we want an UI to be able to benefit from this scrollbar. For this approach restraining the `*` matcher to the root of this wrapper component could make sense
2. Introduce a theme component wrapper that, when introduced in the app, define css variables (https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) for each of the theme color. We then would be able to use those css variables directly in `index.css` 

IMO the first one will be easier to maintain and brings some benefits (decoupling the custom navbar to anything else) despite forcing us to add a wrapper in the UI where we want this custom navbar (but I think its ok and makes it explicit if one day we wonder where does this customisation comes from).